src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -15,9 +15,12 @@ import {
   C_METAL,
   C_GREY_2,
   C_GREY_10,
+  C_GHOST,
 } from '#psammead/psammead-styles/src/colours';
+import { ARTICLE_PAGE } from '#app/routes/utils/pageTypes';
```

Comment:
Just for consistency it may be better to keep these as relative paths instead of aliases. 

I'm a bit on a fence about it and prefer aliasing when using TS components, as TS resolves paths much better than our custom directory aliasing config. A discussion for another time perhaps 😄